app/objects/c_operation.py

```diff
@@ -245,35 +247,32 @@ def report(self, file_svc, output=False, redacted=False):
 
     """ PRIVATE """
 
-    async def _run_phases(self, planner):
-        for phase in self.adversary.phases:
+    async def _run(self, planner):
+        # The following is scope manipulation to force multiple loops of increasing sets of steps for atomic
+        # while forcing a single loop for a non-atomic
```

Comment:
I think if we can rename our variables in this function, we can better express what is happening in here instead of using the comment. I had to read the code a few times to ensure I got it-- but I think just a few renames would do the trick